components/wb-data-json/index.json-ld

```diff
@@ -22,34 +22,34 @@
 			{
 				"title": "Data JSON",
 				"language": "en",
-				"path": "data-json-en.html"
+				"path": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/wb-data-json/data-json-en.html"
```

Comment:
Since the links are wrongly generated by the script in both index pages (en/fr) the parameter "path" must be replaced by "url" from line 25 to 52. The "path" parameter is used for internal links inside GCWeb and "url" for external links.

Reply:
fixed